src/operators.jl

```diff
@@ -571,3 +569,55 @@ function Base.complex(
 )
     return r + im * i
 end
+
+function Base.:+(
```

Comment:
What's wrong with the default implementation of `LinearAlgebra` ? Add a comment that the fault returns a `Matrix` and we overload to return a `Hermitian`